### PR TITLE
9-Cooper

### PR DESCRIPTION
**Pull request by chanson02**
Modified my arrays into dictionaries
https://trello.com/c/dkePfomw/9-modify-array-to-dictionary

### CODE REVIEW

File: plotDamage.py [hunk 1]
```diff
@@ -78,18 +78,52 @@ def plotDamage(attacker: list, defender: list) -> list:
...
-defense2 = ["bubble spell", 3, 3, 3]
-
-attack_profile = [attack1, attack2]
-defense_profile = [defense1, defense2]
-meters = [50, 50, 50]
-
-player1 = [meters, attack_profile, defense_profile]
-player2 = [meters, attack_profile, defense_profile]
-print(plotDamage(player1, player2))
+# print(plotDamage(player1, player2))
```
Comment (NoahBartell) [inline plotDamage.py]: Unused comment

File: plotDamage.py [hunk 2]
```diff
@@ -78,18 +78,52 @@ def plotDamage(attacker: list, defender: list) -> list:
...
+        player_meters,
+        attacks,
+        defenses
+        )
+player2 = create_player(
+        player_meters,
+        attacks,
+        defenses
+        )
+
```
Comment (NoahBartell) [inline plotDamage.py]: Should double check naming conventions, I think methods should be PascalCase and variables camelCase

**Review by NoahBartell — CHANGES_REQUESTED** [review]
Just double check for formatting methods and variables

**Review by MLHumphries — CHANGES_REQUESTED** [review]
Same comment as Noah. PascalCase for methods/functions.

File: plotDamage.py (hunk 2 above)
Comment (chanson02) [reply plotDamage.py]: whoever came up with that rule is wrong.

**Review by MLHumphries — APPROVED** [review]

**Review by NoahBartell — APPROVED** [review]
Lgtm

**Pull request merged**